rats-tls/Makefile

```diff
@@ -0,0 +1,20 @@
+# Root directory of the project (absolute path).
+ROOTDIR=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+RATS_TLS_VERSION := $(shell cat $(ROOTDIR)/VERSION)
+RATS_TLS_MAINTAINER := $(shell head -1 $(ROOTDIR)/MAINTAINERS)
+
+clean:
+	@rm -rf /usr/local/lib/rats-tls
+	@rm -f dist/rpm/rats_tls.spec dist/deb/debian/changelog
+
+install:
+	cmake -DBUILD_SAMPLES=on -H. -Bbuild
+	make -C build install
+
+uninstall:
+	@rm -rf /usr/local/include/rats-tls/
```

Comment:
Please use global env (such as RATS_LTB) to refer to `/usr/local/include/rats-tls/`